androguard/cli/main.py

```diff
@@ -423,6 +423,7 @@ def androsign_main(args_apk, args_hash, args_all, show):
     import traceback
     from colorama import Fore, Style
     from asn1crypto import x509, keys
+    from oscrypto import asymmetric
```

Comment:
please add oscrypto to the requirements.txt

Reply:
will do